flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobCacheCleanupTest.java

```diff
@@ -317,20 +303,21 @@ public void testPermanentBlobDeferredCleanup() throws IOException, InterruptedEx
     }
 
     @Test
-    public void testTransientBlobNoJobCleanup() throws Exception {
-        testTransientBlobCleanup(null);
+    void testTransientBlobNoJobCleanup() throws Exception {
+        testTransientBlobCleanup(tempDir, null);
     }
 
     @Test
-    public void testTransientBlobForJobCleanup() throws Exception {
-        testTransientBlobCleanup(new JobID());
+    void testTransientBlobForJobCleanup() throws Exception {
+        testTransientBlobCleanup(tempDir, new JobID());
     }
 
     /**
      * Tests that {@link TransientBlobCache} cleans up after a default TTL and keeps files which are
      * constantly accessed.
      */
-    private void testTransientBlobCleanup(@Nullable final JobID jobId) throws Exception {
+    private void testTransientBlobCleanup(final Path tempDir, @Nullable final JobID jobId)
```

Comment:
Why do we need to add `Path tempDir` parameter?

Reply:
We don't, I think this was the first test and I went with the temp dir as a method param the first time, just forgot to adapted this part.